state/application.go

```diff
@@ -1459,6 +1459,9 @@ type SetCharmConfig struct {
 	// EndpointBindings is an operator-defined map of endpoint names to
 	// space names that should be merged with any existing bindings.
 	EndpointBindings map[string]string
+
+	// Series, if set, updates the application's series.
+	Series string
```

Comment:
Can we move this attribute up next to where ForceSeries is declared.
We need to update the ForceSeries logic as well.

Reply:
Moved. This series is specified by the charm bases, and the charm pod is blown away and reinstated, so I'm not sure what it would mean to force the series in this case?